src/Symfony/Component/Console/Tests/Input/ArgvInputTest.php

```diff
@@ -238,6 +237,13 @@ public function testParseArrayOption()
         $this->assertSame(array('name' => array('foo', 'bar', null), 'anotherOption' => true), $input->getOptions(), '->parse() parses empty array options as null ("--option value" syntax)');
     }
 
+    public function testParseArrayValueIsCommaSeparatedOption()
+    {
+        $input = new ArgvInput(array('cli.php', '--name=foo,bar'));
+        $input->bind(new InputDefinition(array(new InputOption('name', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY | InputOption::VALUE_IS_COMMA_SEPARATED))));
+        $this->assertSame(array('name' => array('foo', 'bar')), $input->getOptions(), '->parse() parses empty array options as null ("--option=value" syntax)');
```

Comment:
What about:

```
->parse() parses comma separated array options ("--option=value" syntax)
```

?

(I think there was some mistakes in some other assertions, like in l.230 where it should be `->parse() parses array options ("--option=value" syntax)`)